install_config/cluster_metrics.adoc

```diff
@@ -583,9 +453,10 @@ The following command sets the Hawkular Metrics route to use
 You must have a persistent volume of sufficient size available.
 
 ----
-$ oc new-app --as=system:serviceaccount:openshift-infra:metrics-deployer \
-    -f metrics-deployer.yaml \
-    -p HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com
+$ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/common/openshift-cluster/openshift_metrics.yml \
+   -e openshift_metrics_install_metrics=True \
+   -e openshift_metrics_hawkular_hostname=hawkular-metrics.example.com \
+   -e openshift_metrics_cassandra_storage_type=pv
 ----
```

Comment:
For origin we need to provide a link to where we can download the file if OpenShift ansible is not installed (which is a valid use case for origin installations).

For OCP, the location should be known before hand, why its it <OPENSHIFT_ANSIBLE_DIR>? or is this a resolved to something during the build?